src/game/common/multiplayersettings.h

```diff
@@ -69,13 +77,65 @@ class MultiplayerSettings : public SubsystemInterface
 public:
     MultiplayerSettings();
 
+#ifdef GAME_DLL
+    MultiplayerSettings *Hook_Ctor() { return new (this) MultiplayerSettings(); }
+#endif
+
     // Subsystem interface implementation.
+    virtual ~MultiplayerSettings() override {}
     virtual void Init() override {}
     virtual void Reset() override {}
     virtual void Update() override {}
 
-    MultiplayerColorDefinition *Find_Color_Definition(Utf8String name);
-    MultiplayerColorDefinition *New_Color_Definition(Utf8String name);
+    MultiplayerColorDefinition *Find_Multiplayer_Color_Definition_By_Name(Utf8String name);
+    MultiplayerColorDefinition *New_Multiplayer_Color_Definition(Utf8String name);
+    void Add_Starting_Money_Choice(const Money &money, bool is_default);
+
+    static void Parse_Multiplayer_Starting_Money_Choice_Definition(INI *ini);
+    static void Parse_Multiplayer_Settings_Definition(INI *ini);
+    static const FieldParse *Get_Field_Parse() { return s_multiplayerSettingsFieldParseTable; }
+
+    int Get_Max_Beacons_Per_Player() const { return m_maxBeaconsPerPlayer; }
+    const std::vector<Money> *Get_Money_Vector() const { return &m_moneyVec; }
+    int Get_Start_Countdown_Timer() const { return m_startCountdownTimer; }
+    bool Is_Use_Shroud() const { return m_useShroud; }
+    bool Is_Show_Random_Player_Template() const { return m_showRandomPlayerTemplate; }
+    bool Is_Show_Random_Color() const { return m_showRandomColor; }
+    bool Is_Show_Random_Start_Pos() const { return m_showRandomStartPos; }
+
+    int Get_Num_Colors()
```

Comment:
Can be `const`

Reply:
It can't be since it changes a member variable.